src/main/java/com/github/teamfusion/spyglassplus/client/ClientEvents.java

```diff
@@ -0,0 +1,28 @@
+package com.github.teamfusion.spyglassplus.client;
+
+import com.github.teamfusion.spyglassplus.SpyglassPlus;
+import com.github.teamfusion.spyglassplus.client.model.SmallSpyglassStandModel;
+import com.github.teamfusion.spyglassplus.client.model.SpyglassStandModel;
+import com.github.teamfusion.spyglassplus.client.render.SpyglassStandRender;
+import com.github.teamfusion.spyglassplus.core.registry.SpyglassPlusEntityTypes;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+import net.minecraftforge.client.event.EntityRenderersEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@OnlyIn(Dist.CLIENT)
```

Comment:
Given that you use `value = Dist.CLIENT`, the OnlyIn annotation is unnecessary here